extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -22,10 +22,12 @@ import {
 import {addAttributesToElement} from '../../../../../src/dom';
 import {dict} from '../../../../../src/utils/object';
 import {
+  getDocLocation,
```

Comment:
Re: let's not write code that only exists to write unit tests. 99% of the time there's a better solution